CHANGELOG.rst

```diff
@@ -6,7 +6,8 @@
 * Text documents without any doctests no longer appear as "skipped".
   Thanks `@graingert`_ for reporting and providing a full PR (`#1580`_).
 
-*
+* Fix internal error issue when `method` argument is missing for 
+  `teardown_method`. Fixes (`#1605`).
```

Comment:
ok, I'm not really used to ReST, got a strong preference for markdown syntax. BTW, I mimicked the syntax around.